src/gdal.mk

```diff
@@ -82,8 +80,9 @@ define $(PKG)_MAKE
     $(MAKE) -C '$(1)/gcore' -j '$(JOBS)' install
     $(MAKE) -C '$(1)/frmts' -j '$(JOBS)' install
     $(MAKE) -C '$(1)/alg'   -j '$(JOBS)' install
-    $(MAKE) -C '$(1)/ogr'   -j '$(JOBS)' install OGR_ENABLED=
-    $(MAKE) -C '$(1)/apps'  -j '$(JOBS)' install BIN_LIST=
+    $(MAKE) -C '$(1)/ogr'   -j '$(JOBS)' install #OGR_ENABLED=
+    $(MAKE) -C '$(1)/apps'  -j '$(JOBS)' install
+    $(MAKE) -C '$(1)'       -j '$(JOBS)' install #make install on each dir is required?
```

Comment:
It is not required from a quick look, but the original intention is not to install every single directory. Also, leave the `BIN_LIST=` alone as MXE doesn't build applications.